models/logs.js

```diff
@@ -36,7 +36,7 @@ const addLog = async (type, meta, body) => {
  */
 const fetchLogs = async (query, param) => {
   try {
-    const { dev, ...remainingQuery } = query;
+    const remainingQuery = query;
```

Comment:
Do we still need remainingQuery variable?

Reply:
Fixed

Reply:
fixed